tools/doc/html.js

```diff
@@ -518,7 +518,7 @@ function getId(text) {
   text = text.replace(/^_+|_+$/, '');
   text = text.replace(/^([^a-z])/, '_$1');
   if (idCounters.hasOwnProperty(text)) {
-    text += '_' + (++idCounters[text]);
+    text += `_${(++idCounters[text])}`;
```

Comment:
A nit (this can be done on landing): parentheses seem redundant now, this can simply be:
```js
    text += `_${++idCounters[text]}`;
```

Reply:
Sorry, I missed this comment while landing. Doesn't seem worth force-pushing or doing an additional commit at this point to me, although if someone opens a PR for it, sure, I'm OK with that.

Reply:
[Done.](https://github.com/nodejs/node/pull/16845)